websocket/jetty10/src/main/java/jenkins/websocket/Jetty10Provider.java

```diff
@@ -147,6 +147,7 @@ public void onWebSocketText(String message) {
             @Override
             public void onWebSocketClose(int statusCode, String reason) {
                 listener.onWebSocketClose(statusCode, reason);
+                sessions.remove(listener);
```

Comment:
How do you know this will be called before https://github.com/jenkinsci/jenkins/blob/77c2450e5c69eb68f7038a66385237c697266788/websocket/jetty10/src/main/java/jenkins/websocket/Jetty10Provider.java#L99-L100?

Reply:
I don't know. Do you have some insight?

Reply:
No. You would need to test it.